### PR TITLE
Implement init-price command

### PR DESCRIPTION
**Pull request by thmzlt**

### CODE REVIEW

File: program_admin/instructions.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 COMMAND_ADD_PRICE = 4
 COMMAND_ADD_PUBLISHER = 5
 COMMAND_DEL_PUBLISHER = 6
+COMMAND_INIT_PRICE = 4
```
Comment (guibescos) [inline program_admin/instructions.py:17]: This needs to be 9

**Review by guibescos — CHANGES_REQUESTED** [review]
How are you planning to make this change on mainnet (given that we'll probably need the change before mainnet is onboarded onto the multisig?)

File: program_admin/instructions.py [hunk 2]
```diff
@@ -188,6 +189,42 @@ def delete_price(
...
+    data = layout.build(
+        dict(
+            version=PROGRAM_VERSION,
+            command=COMMAND_INIT_PRICE,
+            exponent=exponent,
+            type=price_type,
+        )
+    )
+
+    return TransactionInstruction(
```
Comment (guibescos) [inline program_admin/instructions.py:218]: You need to do it with the new interface if you want this to be compatible with the multisig:
```python
permissions_account = PublicKey.find_program_address([b"permissions"], program_id= program_key)[0]

 return TransactionInstruction(
     data=data,
     keys=[
         AccountMeta(pubkey=funding_key, is_signer=True, is_writable=True),
        AccountMeta(pubkey=price_key, is_signer=False, is_writable=True),
        AccountMeta(pubkey=permissions_account, is_signer=False, is_writable=False)]
        )
   ```
   Basically there's an extra account that contains the admin permission, and `price_key` no longer needs to sign

File: program_admin/instructions.py (hunk 1 above)
Comment (thmzlt) [reply program_admin/instructions.py:17]: I even put it in the right order. I think I hit undo by accident.

**Comment by thmzlt** [comment]
> How are you planning to make this change on mainnet (given that we'll probably need the change before mainnet is onboarded onto the multisig?)

Good point. Let me change this so there's also the option of submitting the transaction directly from program-admin.

**Pull request closed without merging**